.github/workflows/secrel.yml

```diff
@@ -18,6 +18,11 @@ env:
   # while secrets.ACCESS_TOKEN has `repo` and `write:packages` permission
   GITHUB_ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN_GRADLE_BUILD }}
 
+  # Id for the #benefits-vro-devops Slack channel
+  SLACK_CHANNEL: C04CA47HV96
+  DEPLOY_STATE_EMOJI: 'x'
+  JOB_STATUS: 'failed'
```

Comment:
do we need these two env vars?
```
DEPLOY_STATE_EMOJI: 'x'
JOB_STATUS: 'failed'
```
it looks like they're not used right now!

Reply:
unless they're used in another job that's not in this file...

Reply:
They are not! Good call, removed now.